libminifi/src/controllers/SSLContextService.cpp

```diff
@@ -196,16 +196,16 @@ bool SSLContextService::configure_ssl_context(SSL_CTX *ctx) {
   }
 
   // Security level set to 0 for backwards compatibility to support TLS versions below v1.2
-  SSL_CTX_set_security_level(ctx, 0);
+  if ((minimum_tls_version_ != 0 && minimum_tls_version_ < TLS1_2_VERSION) || (maximum_tls_version_ != -1 && maximum_tls_version_ < TLS1_2_VERSION)) {
```

Comment:
should this be `if ((minimum_tls_version_ != -1 && ...`?

Reply:
Yes, good catch, fixed in 8f8721b1712189699a0c4af09e0a06bd3fe763ee